README.rst

```diff
@@ -103,130 +103,132 @@ Functionality Overview
+containers and to/from external services. The load balancing is implemented
+in eBPF using efficient hashtables allowing for almost unlimited scale.
+
+* **East-west load balancing** rewrites service connections at the socket
+  level (``connect()``), avoiding the overhead of per-packet NAT and fully
+  `replacing kube-proxy <https://cilium.io/use-cases/kube-proxy/>`_.
+
+* **North-south load balancing** supports XDP for high-throughput scenarios
+  and `layer 4 load balancing <https://cilium.io/use-cases/load-balancer/>`_
+  including Direct Server Return (DSR), and Maglev consistent hashing.
+
+Cluster Mesh
+------------
+
+Cilium `Cluster Mesh <https://cilium.io/use-cases/cluster-mesh/>`_ enables
+secure, seamless connectivity across multiple Kubernetes clusters. For
+operators running hybrid or multi-cloud environments, Cluster Mesh ensures
+a consistent security and connectivity experience.
+
+* **Global service discovery**: Workloads across clusters can discover and
+  connect to services as if they were local. This enables fault tolerance,
+  like automatically failing over to backends in another cluster, and
+  exposes shared services like logging, auth, or databases across
+  environments.
+
+* **Unified identity model:** Security policies are enforced based on
+  identity, not IP address, across all clusters.
+
+Network Policy
+--------------
+
+Cilium `Network Policy <https://cilium.io/use-cases/network-policy/>`_
+provides identity-aware enforcement across L3-L7. Typical container
+firewalls secure workloads by filtering on source IP addresses and
+destination ports. This concept requires the firewalls on all servers to be
+manipulated whenever a container is started anywhere in the cluster.
+
+In order to avoid this situation which limits scale, Cilium assigns a
+security identity to groups of application containers which share identical
+security policies. The identity is then associated with all network packets
+emitted by the application containers, allowing to validate the identity at
+the receiving node.
+
+* **Identity-based security** removes reliance on brittle IP addresses.
+
+* **L3/L4 policies** restrict traffic based on labels, protocols, and ports.
+
+* **DNS-based policies:** Allow or deny traffic to FQDNs or wildcard domains
+   (e.g., ``api.example.com``, ``*.trusted.com``). This is especially useful
+   for securing egress traffic to third-party services.
+
+* **L7-aware policies** allow filtering by HTTP method, URL path, gRPC call,
+  and more:
+
+  * Example: Allow only GET requests to ``/public/.*``.
+
+  * Enforce the presence of headers like ``X-Token: [0-9]+``.
+
+CIDR-based egress and ingress policies are also supported for controlling
+access to external IPs, ideal for integrating with legacy systems or
+regulatory boundaries.
+
+Service Mesh
+------------
+
+With Cilium `Service Mesh <https://cilium.io/use-cases/service-mesh/>`_,
+operators gain the benefits of service mesh, encryption, observability,
```

Comment:
"Service Mesh ... gains the benefits of service mesh"? Maybe we can substitute the latter `service mesh` for another feature or drop it.